src/components/Header.tsx

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, {
+  useEffect, useRef, useState,
+} from 'react';
+import classNames from 'classnames';
+
+import { Todo } from '../types/Todo';
+import { USER_ID } from '../utils/constants';
+import { useTodo } from '../hooks/useTodo';
+
+type Props = {
+  activeTodos: Todo[];
+  setTempTodo: (todo: Todo | null) => void;
+};
+
+export const Header: React.FC<Props> = ({ activeTodos, setTempTodo }) => {
+  const [title, setTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
+  const [isSubmiting, setIsSubmiting] = useState(false);
+  const todoTitleField = useRef<HTMLInputElement>(null);
+  const {
+    todos,
+    setErrorMessage,
+    handleAddTodo,
+    handleUpdateTodo,
+  } = useTodo();
+
+  useEffect(() => {
+    if (todoTitleField.current) {
+      todoTitleField.current.focus();
+    }
+  });
```

Comment:
`useEffect` without a dependency array, is a bad practice

Currently, this useEffect triggers on every `Header` render, try to look at how it works on the [mate demo](https://mate-academy.github.io/react_todo-app-with-api/). 

I think there they focus input only after delete or add todo (after changing the amount of todos)